ci/scripts/install_kartothek.sh

```diff
@@ -28,9 +28,7 @@ karthothek=$1
 target=$2
 
 git clone --recurse-submodules https://github.com/JDASoftwareGroup/kartothek "${target}"
-if [ "${kartothek}" = "master" ]; then
```

Comment:
Did you check that no existing CI job was passing "master" to this script?

Reply:
I see, it's simply redundant with the `else` branch below?

Reply:
Correct

Reply:
Yes, we can safely remove this branch.